ScribusGeneratorBackend.py

```diff
@@ -65,7 +65,34 @@ def run(self):
 
         #generating
         logging.debug("parsing data source file %s"%(self.__dataObject.getDataSourceFile()))
-        csvData = self.getCsvData(self.__dataObject.getDataSourceFile())        
+        csvData = self.getCsvData(self.__dataObject.getDataSourceFile())
+        if(len(csvData) < 1):
+            logging.error("Data file %s is empty. At least a header line and a line of data is needed. Halting."%(self.__dataObject.getDataSourceFile()))

```

Comment:
logging is not enough when the "error" or mis-use is likely to happen, need to feedback to the user frontend (CLI and GUI).